content/en-us/mine/khala-mining/soloError Summary about Mining&Delegation.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Error Summary about Mining & Delegation"
-weight: 1020
+weight: 1010
 menu:
   mine:
     parent: "mine-solo"
```

Comment:
If you want to move this to `Mine Khala` section, you need to change this to `parent: "khala-mining"`.
You can find the identifier for each section in <https://github.com/Phala-Network/phala-wiki-next/blob/master/config/_default/menus/menus.en-us.toml>.